core-bundle/src/Resources/contao/library/Contao/Environment.php

```diff
@@ -314,7 +314,7 @@ protected static function path(Request $request)
 	 */
 	protected static function script()
 	{
-		return preg_replace('/^' . preg_quote(static::get('path'), '/') . '\/?/', '', static::get('scriptName'));
+		return static::get('scriptName');
```

Comment:
```suggestion
		trigger_deprecation('contao/core-bundle', '5.0', 'Using "%s()" has been deprecated and will no longer work in Contao 6.0. Use "%s::%s" instead.', __METHOD__, __CLASS__, 'scriptName');

		return preg_replace('/^' . preg_quote(static::get('path'), '/') . '\/?/', '', static::get('scriptName'));
```
I think we should not change these methods, but instead deprecate them and replace all usages of `Environment::get('script')` with `Environment::get('scriptName')`